src/Symfony/Bundle/SecurityBundle/Tests/Functional/Bundle/GuardedBundle/AuthenticationController.php

```diff
@@ -33,6 +33,6 @@ public function profileAction(UserInterface $user = null)
             return new Response('Not logged in.');
         }
 
-        return new Response('Username: '.$user->getUsername());
+        return new Response('Username: '.$user->getUserIdentifier());
```

Comment:
Shall we keep the username string here?

Reply:
Yes, I think it's fine. The identifier can be the username (and anyway, this is just some test response).